index.js

```diff
@@ -10,9 +13,11 @@ mongoose
     .then(() => console.log('DB OK'))
     .catch((err) => console.log('DB ERROR', err))
 
-app.get('/', (req, res) => {
-    res.send('Working');
-})
+app.post('/auth/register', registerValidation, register);
+
+app.post('/auth/login', loginValidation, login);
+
+app.get('/auth/me', checkAuth, getMe);
```

Comment:
After making general imports:
```suggestion
app.post('/auth/register', registerValidation, UserController.register);
app.post('/auth/login', loginValidation, UserController.login);
app.get('/auth/me', checkAuth, UserController.getMe);

```